changelogs/unreleased/4361-the-set-setting-endpoint-return-status-code-500-on-invalid-value.yml

```diff
@@ -0,0 +1,5 @@
+description: The set_setting endpoint now correctly returns a 400 status code when an invalid value is provided.
+change-type: patch
+destination-branches: [master, iso4, iso5]
+issue-nr: 4361
+sections: {}
```

Comment:
Is this worthy of a bugfix entry ?

Reply:
Yes. This changes the behavior of our API, so that's definitely relevant to end-users.